nix/dpdk/default.nix

```diff
@@ -98,13 +98,13 @@ stdenv.mkDerivation rec {
     "reorder"
     "rib"
     "sched"
-    "security"
     "table"
   ];
 
   enabledLibs = [
     "timer" # needed for i40e
     "eventdev" # needed for i40e
+    "security" # needed for i40e
```

Comment:
Commit description is a copy/paste from previous commit about `eventdev`. Please replace with `security` for this commit.

\+ typo: `stadly`.